### PR TITLE
[ar] Translate home/supported-doc-versions.md

### PR DESCRIPTION
**Pull request by adowair**
This PR will create a new file `contents/docs/home/supported-doc-versions.md`,
containing a translation of the corresponding content into Arabic.

This PR contributes to achieving #44682 and #37546.

### CODE REVIEW

**Comment by adowair** [comment]
@mboukhalfa Here is a draft PR to hopefully chip away at this task. First of many!

@sftim in response to your comment here https://github.com/kubernetes/website/pull/37547#issuecomment-1293263536,
would you be willing to create a new development branch for us to target 1.29?

Thank you!

File: content/ar/docs/home/supported-doc-versions.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+card:
+  name: about
+  weight: 10
+  title: إصدارات التوثيق المتوفرة
+---
+
+يوثقُ هذا الموقع الإصدار الحالي والإصدارات الأربع السابقة من كوبرنيتيس.
+
+لا يرتبط وجود توثيقٍ لإصدارٍ معينٍ بالدعم الرسمي لهذا الإصدار.
+تعَرَّف على الإصدارات المدعومة، ومدةِ دعمها، على صفحة [مُدَّة الدعم](/releases/patch-releases/#support-period).
```
Comment (adowair) [inline content/ar/docs/home/supported-doc-versions.md:14]: This link may need to have the `ar` subdirectory prepended.
```suggestion
تعَرَّف على الإصدارات المدعومة، ومدةِ دعمها، على صفحة [مُدَّة الدعم](/releases/patch-ar/releases/#support-period).
```

**Comment by mboukhalfa** [comment]
Thanks for the great work! we will have a branch to collect those PR soon so we can have the base changes needed to see the translated text in the deploy preview. I will setup this soon with the community. Please keep opening PRs and we will merge them once the branch is created

**Comment by adowair** [comment]
Closing this PR to reopen with these changes elsewhere...

**Pull request closed without merging**